dhis-2/dhis-api/src/main/java/org/hisp/dhis/programrule/ProgramRuleAction.java

```diff
@@ -326,8 +338,25 @@ public String getTemplateUid() {
     return templateUid;
   }
 
-  public void setTemplateUid(String programNotificationTemplate) {
-    this.templateUid = programNotificationTemplate;
+  public void setTemplateUid(String templateUid) {
```

Comment:
What happens if both templateUid and programNotificationTemplate are defined and they have different values?
Who is the winner?

Reply:
`programNotificationTemplate` will be ignored if it's there.

Reply:
we are not changing API.